metamodel-begrippen.md

```diff
@@ -182,107 +182,107 @@ Gerelateerd: [Domeinobject](#domeinobject)
 
 Specialisatie van: [Eigenschap](#eigenschap)
 
 Gerelateerd: [Waarde](#waarde)
 
 ### Lengte
 
-> Een lengte is een conditie waarbij van een kenmerk of waarde is gesteld hoe lang de waarde of invulling van dat kenmerk mag zijn
+> Een lengte is een conditie waarbij van een kenmerk of waarde is gesteld hoe lang de waarde of invulling van dat kenmerk mag zijn.
 
-Toelichting: Een lengte kan zowel op een specifiek kenmerk van toepassing zijn (bijvoorbeeld de lengte die een voornaam-kenmerk mag hebben), maar ook voor gelijksoortige waarden (bijvoorbeeld de lengte die een postcode-waarde mag hebben)
+Toelichting: Een lengte kan zowel op een specifiek kenmerk van toepassing zijn (bijvoorbeeld de lengte die een voornaam-kenmerk mag hebben), maar ook voor gelijksoortige waarden (bijvoorbeeld de lengte die een postcode-waarde mag hebben).
 
 Specialisatie van: [Conditie](#conditie)
 
 ### Letterlijke waarde
 
-> Een letterlijke waarde is een waarde waarvan de betekenis letterlijk genomen moet worden, de waarde zelf en niets meer
+> Een letterlijke waarde is een waarde waarvan de betekenis letterlijk genomen moet worden, de waarde zelf en niets meer.
 
 Specialisatie van: [Waarde](#waarde)
 
 ### Objecttype
 
-> Een objecttype is een typering van gelijksoortige domeinobjecten
+> Een objecttype is een typering van gelijksoortige domeinobjecten.
 
 Gerelateerd: [Domeinobject](#domeinobject)
 
 ### Populatie
 
-> Een populatie is de verzameling van alle mogelijke domeinobjecten die te onderscheiden zijn als objecttype
+> Een populatie is de verzameling van alle mogelijke domeinobjecten die te onderscheiden zijn als objecttype.
 
 Gerelateerd: [Objecttype](#objecttype), [Domeinobject](#domeinobject)
 
 ### Relatie
 
-> Een relatie is een verbintenis tussen domeinobjecten
+> Een relatie is een verbintenis tussen domeinobjecten.
 
 Gerelateerd: [Domeinobject](#domeinobject)
 
 ### Relatiedomeinobject
 
-> Een relatiedomeinobject is een relatie die beschouwd wordt als domeinobject
+> Een relatiedomeinobject is een relatie die beschouwd wordt als domeinobject.
 
 ### Relatieobjecttype
 
-> Een relatieobjecttype is een typering van gelijksoortige relatiedomeinobjecten
+> Een relatieobjecttype is een typering van gelijksoortige relatiedomeinobjecten.
 
 Gerelateerd: [Relatiedomeinobject](#relatiedomeinobject)
 
 ### Relatietype
 
-> Een relatietype is een typering van gelijksoortige relaties
+> Een relatietype is een typering van gelijksoortige relaties.
 
 ### Rol
 
-> Een rol is een eigenschap van een domeinobject in een relatie met zichzelf of een ander domeinobject
+> Een rol is een eigenschap van een domeinobject in een relatie met zichzelf of een ander domeinobject waarmee een functie, taak of gedragspatroon die dat domeinobject (zoals een object, systeem, dier, softwarecomponent, robot, enz.) vervult binnen een bepaald systeem of situatie wordt gekenmerkt.
```

Comment:
Het is inderdaad wel een verbetering, omdat de definitie hiermee onderscheidender is. Echter, in de definitie worden nu voorbeelden genoemd van domeinobjecten en dat hoort niet in de definitie van rol thuis (mag wel in de toelichting). En ik zou ik niet "in een bepaald systeem of situatie" stellen, maar juist "met betrekking tot de betreffende relatie". Dus iets als ", waarmee een functie, taak of gedrag van die dat domeinobject vervult binnen de relatie wordt gekenmerkt".

Reply:
Ik heb de voorbeelden nu weggelaten aangezien het onderwerp van deze paragraaf 'Rol' is en niet 'Domeinobject'. Voorbeelden van domeinobjecten hoeven n.m.m. dan ook niet hier genoemd te worden.